games/static/games/proto.js

```diff
@@ -0,0 +1,97 @@
+class Square{
+    constructor(indexOnBoard, div) {
+        this.index = indexOnBoard
+        this.div = div
+    }
+    setCharacter(character) {
+        this.character = character
+        this.div.class = character.getCssClass
```

Comment:
thats a function, call it with `()` :
```
character.getCssClass()
```

Reply:
I found out in the end... the hard way though